core/src/test/java/org/frankframework/jdbc/JdbcQueryListenerTest.java

```diff
@@ -0,0 +1,60 @@
+package org.frankframework.jdbc;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+import javax.sql.DataSource;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import org.frankframework.configuration.ConfigurationException;
+import org.frankframework.dbms.H2DbmsSupport;
+import org.frankframework.receivers.Receiver;
+
+@SuppressWarnings("removal")
+class JdbcQueryListenerTest {
```

Comment:
It's a deprecated class but now at least there is some test-coverage on it outside of the Larva tests.